### PR TITLE
increase table and round temperature readings

### PR DESCRIPTION
**Pull request by kopr12**

### CODE REVIEW

File: sct.c [hunk 1]
```diff
@@ -97,7 +171,7 @@ static int get_sct_for_screen(Display *dpy, int screen)
 
         XFree(crtc_gamma);
     }
-    temp = (int)(ts / (double)n);
+    temp = 100 * round((int)(ts / (double)n) / 100.0);
```
Comment (zvezdochiot) [inline sct.c]:
```suggestion
    temp = (int)(ts / (double)n * 0.01 + 0.5) * 100;
```

**Review by zvezdochiot — COMMENTED** [review]
Why not a complete table with changing limits? (see #7)

**Comment by kopr12** [comment]
I don't think anyone would use temperatures above 10000K, but I agree with why not :)
I accepted your suggestion regarding rounding, nice solution, avoiding `math.h` header and `-lm` in Makefile, there are number of ways to do that, I went with the stupid one.

If you want I can delete this PR and you can make the table changes ?
It's not a problem.

**Comment by zvezdochiot** [comment]
Not. Wait what @faf0 say.

**Comment by zvezdochiot** [comment]
@kajzersoze , since you have a more detailed table, I can offer one more change for the test:

#6 (Approximation or search in a table with parabolic interpolation?)

**Comment by kopr12** [comment]
I think that @faf0  probably likes 500 steps for some reason, the table he started this with.
500 steps doesn't work for me, there's a big difference between them on lower temperatures.
Maybe he thinks if the table is bigger then this little utility is more bloated ?
Until he says something we can only speculate.
Maybe you should make a separate PR with new method for searching the table, approximation with rounding seems to be working fine in my tests though.

**Comment by zvezdochiot** [comment]
@kajzersoze say> 500 steps doesn't work for me, there's a big difference between them on lower temperatures.

Of course does not work:

https://github.com/faf0/sct/blob/a38a0df3f85e0c437de3a28cf1627bcae68ef4ff/sct.c#L112
`ratio == 0`

This is how it will work:
```c
    double ratio = (double)(temp % TEMPERATURE_STEP) / TEMPERATURE_STEP;
```
`ratio != 0`
 
Good luck.

**Comment by zvezdochiot** [comment]
@kajzersoze say> Maybe you should make a separate PR with new method for searching the table.

It's too early. #9 (Approximation or table?)

**Comment by faf0** [comment]
The step size for the original table comes from the code here: https://github.com/faf0/sct/blob/a38a0df3f85e0c437de3a28cf1627bcae68ef4ff/sct.c#L8

Having a larger table is fine. The suggestion from https://github.com/faf0/sct/issues/9 seems nice.

**Comment by kopr12** [comment]
I'll close this one, seems like approximation is the way to go :)

**Pull request closed without merging**